src/main/java/com/snowflake/snowpark_java/Row.java

```diff
@@ -425,6 +425,64 @@ public Row getObject(int index) {
     return (Row) get(index);
   }
 
+  /**
+   * Returns the value at the specified column index and casts it to the desired type {@code T}.
+   *
+   * <p>Example:
+   *
+   * <pre>{@code
+   * Row row = Row.create(1, "Alice", 95.5);
+   * row.getAs(0, Integer.class); // Returns 1 as an Int
+   * row.getAs(1, String.class); // Returns "Alice" as a String
+   * row.getAs(2, Double.class); // Returns 95.5 as a Double
+   * }</pre>
+   *
+   * @param index the zero-based column index within the row.
+   * @param clazz the {@code Class} object representing the type {@code T}.
+   * @param <T> the expected type of the value at the specified column index.
+   * @return the value at the specified column index cast to type {@code T}.
+   * @throws ClassCastException if the value at the given index cannot be cast to type {@code T}.
+   * @throws ArrayIndexOutOfBoundsException if the column index is out of bounds.
+   * @since 1.14.0
```

Comment:
please change this to `1.15.0`

Reply:
Fixed on commit [c8ebed8](https://github.com/snowflakedb/snowpark-java-scala/pull/148/commits/c8ebed8047687ea5de907a3cf9bc5c855b7b76b8)